cli/lib/kontena/plugin_manager.rb

```diff
@@ -167,17 +167,17 @@ def load_plugins
           if File.exist?(plugin) && !plugins.find{ |p| p.name == spec.name }
             begin
               if spec_has_valid_dependency?(spec)
+                ENV["DEBUG"] && $stderr.puts("Loading plugin #{spec.name}")
                 load(plugin)
                 plugins << spec
               else
                 plugin_name = spec.name.sub('kontena-plugin-', '')
                 $stderr.puts " [#{Kontena.pastel.red('error')}] Plugin #{Kontena.pastel.cyan(plugin_name)} (#{spec.version}) is not compatible with the current cli version."
                 $stderr.puts "         To update the plugin, run 'kontena plugin install #{plugin_name}'"
               end
-            rescue LoadError => ex
-              $stderr.puts " [#{Kontena.pastel.red('error')}] Failed to load plugin: #{spec.name}"
+            rescue ScriptError, StandardError => ex
+              warn " [#{Kontena.pastel.red('error')}] Failed to load plugin: #{spec.name}\n\tRerun the command with environment DEBUG=true set to get the full exception.\nYou can uninstall the plugin using: kontena plugin uninstall #{spec.name.sub('kontena-plugin-', '')}\n"
```

Comment:
But if this warning happens, then `kontena plugin uninstall` will not work to uninstall the plugin, because it will be missing from `@plugins` and thus not `installed`.